### ISSUE TITLE
Fix keyboard accessibility

### ISSUE THREAD

**Issue by yileifeng** [ramp4-pcar4/highcharts-editor#37]
## Description

Major keyboard accessibility issues throughout app: 
- [ ] tabbing to disabled side menu option and hitting enter enables it 
- [ ] do not make hidden file upload input focusable
- [ ] make datatable cell values keyboard accessible 
- [ ] tabbing on a column/row action with dropdown opens behaves as enter
- [ ] reduce button padding for focus outline
- [ ] other buttons that are not focusable
- [ ] customization nav bar options not tabbable

## Related issues

https://github.com/ramp4-pcar4/highcharts-editor/issues/30
